--- origsrc/libpqxx-2.6.9/configure.ac	2007-02-01 04:09:44.000000000 -0600
+++ src/libpqxx-2.6.9/configure.ac	2007-08-23 07:27:22.781250000 -0500
@@ -30,7 +30,7 @@
 AC_PROG_CXX
 AC_PROG_CC
 AC_PROG_INSTALL
-AC_DISABLE_SHARED
+AC_DISABLE_STATIC
 AC_PROG_LIBTOOL
 AC_PROG_MAKE_SET
 AC_PATH_PROG([MKDIR], [mkdir])
--- origsrc/libpqxx-2.6.9/src/Makefile.am	2007-01-29 13:18:53.000000000 -0600
+++ src/libpqxx-2.6.9/src/Makefile.am	2007-08-23 07:12:47.734375000 -0500
@@ -26,6 +26,7 @@
 # Uncomment to build a versioned library
 #libpqxx_version = -version-info $(ABI_CURRENT):$(ABI_REVISION):$(ABI_AGE)
 libpqxx_la_LDFLAGS = $(libpqxx_version) \
+	-no-undefined \
 	-rpath $(libdir) \
 	${POSTGRES_LIB}
 
